tests/unit/test_results.py

```diff
@@ -1,21 +1,183 @@
+import unittest
+
 import earthaccess
+import vcr
+from earthaccess.search import DataCollections
+
+my_vcr = vcr.VCR(
+    record_mode="once",
+    decode_compressed_response=True,
+    # Header matching is not set by default, we need that to test the
+    # search-after functionality is performing correctly.
+    match_on=["method", "scheme", "host", "port", "path", "query", "headers"],
+)
```

Comment:
I'm so excited about this. When we want to update a cassette, we just delete it and re-run tests, correct?

Reply:
That is one way to do it. The other is to leverage the record_mode which is currently,
`record_mode="once"`

Reply:
Another area where we have had success with VCR is simulating edge and error cases. You can formulate your own responses from CMR and test how your code responds to it. For example, how does your client handle 500 errors from CMR?